### PR TITLE
Changes in Installation guide - Kibana

### LINKED ISSUE #2275: Installation guide - Kibana improvements
The goal is to test and verify Kibana installation section of a New Documentation Structure.

### PR DESCRIPTION
**Pull request by dkempny**
Hello Team,
this PR closes #2275 

## Description

The modified files include a rephrased, restyled version of Kibana installation guide. Information is added. 

## Checks
- [x] It compiles without warnings.
- [x] Spelling and grammar. 

All the best,
Daria

### CODE REVIEW

File: source/_templates/installations/elastic/common/configure_kibana.rst [hunk 1]
```diff
@@ -4,18 +4,18 @@
...
   # curl -so /etc/kibana/kibana.yml https://raw.githubusercontent.com/wazuh/wazuh/elastic-secured-3.10/extensions/kibana/7.x/kibana.yml
 
-The file ``/etc/kibana/kibana.yml`` has to be edited:
+Edit ``/etc/kibana/kibana.yml`` file:
 
 .. code-block:: yaml
 
     server.host: <kibana_ip>
-    elasticsearch.hosts: https://<elasticsearch_ip>:9200
+    elasticsearch.hosts: ["https://<elasticsearch_ip>:9200"]
```
Comment (alberpilot) [inline source/_templates/installations/elastic/common/configure_kibana.rst]: Are the characters `["` necessaries? If not, the less characters, the lesser user error possibilities.

Comment (dkempny) [reply source/_templates/installations/elastic/common/configure_kibana.rst]: The characters are not necessary if there is only one Elasticsearch node listed.

In elasticsearch documentation it is presented like this:
"**elasticsearch.hosts:**
Default: [ "http://localhost:9200" ] The URLs of the Elasticsearch instances to use for all  your queries. All nodes listed here must be on the same cluster." 

https://www.elastic.co/guide/en/kibana/current/settings.html#settings

Should I change it back?

**Comment by dkempny** [comment]
Discussed changes are now included.

File: source/installation-guide/kibana/index.rst [hunk 2]
```diff
@@ -104,12 +110,51 @@ Kibana installation and configuration
...
+~~~~~~~~~~~~~~~~
+
+To uninstall Kibana:
+
+    .. code-block:: console
+
+      # apt-get remove kibana
 
-Once the Wazuh - Elastic Stack environment is ready, you may install a Wazuh agent in every endpoint to be monitored. The Wazuh installation guide is available for most operating systems and it can be found :ref:`here<installation_agents>`.
+There are files marked as configuration and data files. Due to this designation, the package manager doesn't remove those files from the filesystem. The complete files removal action is a user responsibility. It can be done by removing the folder ``/var/lib/kibana`` and ``/etc/kibana``.
```
Comment (davidfmiranda) [inline source/installation-guide/kibana/index.rst]: I think we should avoid using contractions like  "doesn't" in the documentation.

Comment (davidfmiranda) [inline source/installation-guide/kibana/index.rst]: To uninstall Kibana we should add tabs like the ones used in Elasticsearch and Wazuh sections to include the Yum and Zypp options.  Besides, when using `apt-get remove` with the flag `--purge` all the files marked as configuration and data files will be removed.

Comment (dkempny) [reply source/installation-guide/kibana/index.rst]: Good suggestion.

Comment (dkempny) [reply source/installation-guide/kibana/index.rst]: I will investigate this.

**Pull request merged**